README.md

```diff
@@ -125,17 +125,24 @@ Using local NVD cache as last update was within two hours
 
 ## Known issues
 
-Please see GitHub issues.
+Please see [GitHub issues](https://github.com/SAP/fosstars-rating-core/issues).
 
 ## How to obtain support
 
-Please create a new GitHub issue if you've found a bug, or you'd like to propose an enhancement.
+Please create a new [GitHub issue](https://github.com/SAP/fosstars-rating-core/issues)
+if you found a bug, or you'd like to propose an enhancement.
```

Comment:
Please create a new Github issue if you find a bug or if you'd like to propose an enhancement